Frends.Oracle.ExecuteQuery/Frends.Oracle.ExecuteQuery/Definitions/ConnectionProperties.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.Oracle.ExecuteQuery.Definitions
+{
+    /// <summary>
+    /// Properties to establish connection to the Oracle database.
+    /// </summary>
+    public class ConnectionProperties
+    {
+        /// <summary>
+        /// Oracle connection string.
+        /// </summary>
+        [DisplayFormat(DataFormatString = "Text")]
+        [DefaultValue("Data Source=(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=MyHost)(PORT=MyPort))(CONNECT_DATA=(SERVICE_NAME=MyOracleSID)));User Id=myUsername;Password=myPassword;")]
+        public string ConnectionString { get; set; }
```

Comment:
Connection string is a secret, so you should add `[PasswordPropertyText]`.

Reply:
Changed [DisplayFormat()] to [PasswordPropertyText]